dev/composer-install-verify.php

```diff
@@ -0,0 +1,11 @@
+<?php
+copy('https://getcomposer.org/installer', 'composer-setup.php');
```

Comment:
Does `copy` retry upon a failure? How about to keep the `curl` with the retry option and just verify the hash in this script?